Source/DbgEngHelpers/DebugClient.cs

```diff
@@ -25,6 +25,19 @@ public static IDebugClient OpenDumpFile(string dumpFile, string symbolPath)
             return client;
         }
 
+        public static IDebugClient OpenProcess(string processPath, string processArguments, string symbolPath, uint debugEngineOptions)
+        {
+            string processCommandLine = processPath + " " + processArguments;
+
+            IDebugClient client = DebugCreate();
+
+            ((IDebugSymbols5)client).SetSymbolPathWide(symbolPath);
+            ((IDebugControl7)client).SetEngineOptions(debugEngineOptions);
+            client.CreateProcessAndAttach(0, processCommandLine, 2, 0, 0);
```

Comment:
Solved. Btw, in future CLs we may want to add more granular support for creating new processes. For now I guess we can use this as default.

---
In reply to: [69627279](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r69627279) [](ancestors = 69627279)